markdown/events/2022/hackathon-october-2022.md

```diff
@@ -142,9 +241,112 @@ Keep an eye out for others hosting community events in your area. We will be sha
 
 You can add your community event using [this link](https://seqera.typeform.com/streaming-event).
 
+## Social Activities
+
+During the hackathon, we will have a few light-hearted fun and games!
+
+### On-site
+
+- Monday evening: Beers at Seqera Office!
+
+- Tuesday evening: Dinner!
+
+- The traditional sock hunting in Gathertown will also be held in person! Take pictures of as many socks as you can find.
+
+- Finally, on Thursday, we will be running a short quiz!
+
+### Online
+
+- Throughout the three days, we will once again be running a nf-core hackathon **bingo**! To join the game, you can go the following [link](https://nfcore-bingo.web.app/?game=nf-core-hackathon). Check the instructions at the bottom of the page.
+
+  > <i class="fas fa-hand-paper"></i> Bingo! <https://nfcore-bingo.web.app/?game=nf-core-hackathon>
+
+- In addition, we will be running a sock hunt once more! There are 11 socks distributed around the gather.town world. Take screenshots of as many as you can find!
+- Finally, on Thursday, we will be running a short quiz!
+
+All social activities are of course optional, but hope to see as many people joining in as possible :tada:
+
 # COVID control
 
 Please remember that the COVID-19 pandemic is not over.
 If you intend on joining the hackathon in person, we request that you get vaccinated and tested, if possible. We ask for anyone who displays symptoms to not mingle with other participants. Whilst it is not feasible for us to enforce any rules, we request that attendees behave responsibly.
 
 We will be monitoring the status of the pandemic as the event draws closer and reserve the right to make changes in our policies if we deem that to be necessary.
+
+# Safety Information at the Hackathon
+
+Please note that our [Code of Conduct](https://nf-co.re/code_of_conduct) applies to the Hackathon, and all participants need to abide by our guidelines to participate. We should all feel responsible for making nf-core events safe and fun for everyone.
+
+In addition, please respect the following at all times during the hackathon:
+
+- **Never share the jitsi and zoom links** for the hackathon on social media.
+- Do not take screenshots of groups you are working within without asking for consent of all individuals in the group. Remember, not everyone wants their photographs shared on social media. Not respecting the latter amounts to doxing, which is also in direct violation of the nf-core [Code of Conduct](https://nf-co.re/code_of_conduct).
+
+## Reporting CoC Violations during the hackathon
+
+In case of an immediate perceived threat at the hackathon, please reach out to any of the following individuals on Slack. Their names and the time-zones they are located in are indicated.
+
+- Alexander Peltzer (CET)
```

Comment:
Should just be the 3x safety team. No mention of time zones, just say online / in person.